Dockerfile.fips

```diff
@@ -0,0 +1,52 @@
+# This version of Go is a Go+BoringCrypto release for FIPS 140-2 compliance
+FROM us-docker.pkg.dev/google.com/api-project-999119582588/go-boringcrypto/golang:1.18.5b7 as build-fips
+WORKDIR /src
+COPY go.mod go.sum /src/
+RUN go mod download -x
+COPY . .
+RUN rm -rf /src/webconsole/dist
+RUN GOOS=linux GOARCH=amd64 WEBCONSOLE=default make immuadmin-static immudb-fips
+RUN mkdir /empty
+
+### distroless FIPS 140-2
+FROM gcr.io/distroless/base:nonroot AS distroless-fips
+LABEL org.opencontainers.image.authors="Codenotary Inc. <info@codenotary.com>"
+
+ARG IMMU_UID="65532"
```

Comment:
Other dockerfiles use 3322 UID/GID, is it safer to use larger values ? I could see a potential issue here if someone would like to easily switch between fips and non-fips image. You would have to assume different UID/GID for both images.